src/app/main/components/movies-view/movies-view.component.ts

```diff
@@ -14,8 +14,9 @@ import { MoviesService } from 'src/app/main/services/movies.service';
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
 export class MoviesViewComponent implements OnInit {
-  public movies = new BehaviorSubject<Movie[]>([]);
+  public movies$ = new BehaviorSubject<Movie[]>([]);
```

Comment:
Please strictly type the component fields. They are not readonly and can be assigned any value